src/odemis/util/test/util_test.py

```diff
@@ -144,6 +145,32 @@ def test_simple(self):
             o = util.almost_equal(*i)
             self.assertEqual(o, eo, "Failed to get correct output for %s" % (i,))
 
+
+class PerpendicularDistanceTestCase(unittest.TestCase):
+
+    def test_simple(self):
+        a = (0, 0)
+        b = (1, 0)
+        # Follow a segment at 1 unit away
+        for e in ((0, 1), (0.5, 1), (1, 1)):
+            dist = perpendicular_distance(a, b, e)
+            self.assertAlmostEqual(dist, 1)
+
+        # Follow a segment on the original segment itselft => dist == 0u
```

Comment:
point or segment?

Reply:
It's a point... which moves along a line segment.
=> I've updated the comment to make it clearer.